### ISSUE TITLE
✨ Update a connector

### ISSUE THREAD

**Issue by tjarbo** [tjarbo/discord-moodle-bot#145]
> **As a** user
> **I want** want to change the configuration of a connector 
> **because** e.g the discord channel changed

> ⚠️ Blocked by #144

## Description:
With #144 a user can see a list of all connectors. Now it's time that the user can change the configuration. The following attributes can be changed:
* `active`
* `default`
* `socket` 

It should be not possible to change any other attribute! 

### 🟢 In scope:
* Modular input validation!
* More input validation!
* Implement `PATCH /api/connectors/:id`
* Show notification when a connector has been updated (with name!)

### 🔴 Not in scope:
* Implement other endpoints
* Update Documentation
